### ISSUE TITLE
Verify `attributes` and `miscselectMask` of the quoting enclave

### ISSUE THREAD

**Issue by Niederb** [integritee-network/pallets#127]
- The two attributes of the quoting enclave must be compared against the registered quoting enclave
  - A binary mask must be applied first before comparing
  - All the data is already there

**Comment by clangenb** [comment]
From the code I assume that we don't preform this check yet?

**Comment by Niederb** [comment]
Yes, this is not implemented yet. I did this on purpose because I thought it can be a good first issue for someone else to get into the code.
